### DISCUSSION TITLE
Suggested QuillPad features

### DISCUSSION THREAD

**Discussion by deeeseee** [quillpad/quillpad#189] [Ideas]
Hi all,

I've recently been looking for a FOSS app as an alternative to ColorNote, and Quillpad is the one that came closest to my needs. It appears to be a beautiful and functional app! However, I feel a couple of features would help immensely:

1. The website mentions "Search through notes" as a feature. However, if I'm in a long note or checklist, I can't find any way to search through an individual note. For example, let's say I have a 200-item shopping list with 10 items unchecked for future required purchases. If I run out of eggs, I'd like to be able to quickly search for "Eggs" inside the Shopping checklist. Currently, I can search for "Eggs" in the main app, which will find the Shopping list, but it won't take me to Eggs inside the Shopping list, so that I can uncheck Eggs.

2. The feature where checked items on the checklist automatically move to the bottom of the list is great! However, to be able to reuse a long checklist again in the future, I need to be able to select all items, and then uncheck all of them simultaneously. Also, there has to be a way to sort the list into Alphabetical order.

I really love the app, and hope to start using it for my family devices! But I hope you might consider adding the above features, if it's not too difficult. I found no other app with the same combination of features as Quillpad, so thank you for all your work on this app!

**Comment by clach04** [comment]
Number 1 appears to be a request for https://github.com/quillpad/quillpad/issues/74

**Comment by tonikeys** [comment]
Voting for point 2, extremely needed feature

**Comment by aabraxas** [comment]
Feature suggestions:

Heat map / calendar
for orientation and visible usage would be a helpful feature

Backup via md files: Saving pictures, audio and other attachments in a folder that can be specified by the user.
This feature is very useful since all the links to attached files can be preserved with the folder structure when copied as md and folder structures.

**Reply by aabraxas** [reply]
At present I go through the following routine:
I have created QuillPad Folder with subfolders for images, audio, video, attachments. I store the file (photo, etc.) I want to use in a note in the subfolder. Following that, I create the note and attach the file from the attachment folder. At least that way I can copy all attachments in an organized way.
